### PR TITLE
Precommit additional fixes

### PR DESCRIPTION
**Pull request by benjwadams**

### CODE REVIEW

**Comment by benjwadams** [comment]
@ocefpaf, sort/isort output generated seems to be different than what's outputted by black.   Any suggestions on resolving the outstanding precommit issues?

**Comment by benjwadams** [comment]
pre-commit.ci autofix

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -14,24 +14,19 @@ repos:
...
   hooks:
   - id: isort
     additional_dependencies: [toml]
-    args: [--project=compliance_checker, --multi-line=3, --lines-after-imports=2, --lines-between-types=1, --trailing-comma, --force-grid-wrap=0, --use-parentheses, --line-width=88]
-
-- repo: https://github.com/asottile/seed-isort-config
-  rev: v2.2.0
-  hooks:
-    - id: seed-isort-config
+    args: ["--profile", "black", "--filter-files"]
```
[the remark this answers is not in the document]
Comment (benjwadams) [reply .pre-commit-config.yaml:29]: Commits to the PR are fine.

**Comment by ocefpaf** [comment]
The file sorting is puzzling me. Not sure what is going on there.

**Comment by benjwadams** [comment]
No rush, will be nice to eventually have this in the green.  If we can't figure it out soon, I can merge the current PR and the file sorting can be revisited later.

**Comment by benjwadams** [comment]
Giving these directions a try: https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

**Comment by benjwadams** [comment]
pre-commit.ci autofix

**Comment by benjwadams** [comment]
YESSSS IT WORKS.

The problem was that both `requirements-txt-fixer` and `file-contents-sorter` were sorting `test_requirements.txt`, but had different rules and so were creating an unsolvable pre-commit step.  I've removed `file-contents-sorter` since it's both redundant and causes the pre-commit step to fail.   I'm still not entirely clear how `pre-commit` determines what is a requirements file even after looking at the source, but this appears to work now.

NB: Integration tests are failing due to ERDDAP throttling of IPs -- I think GHA uses the same IP address when running tests?

**Comment by benjwadams** [comment]
Going to go ahead and merge and see if we can't get some green for once on the front page!

**Pull request merged**